api/server.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const helmet = require('helmet');
+const cors = require('cors');
+const session = require('express-session');
+const KnexSessionStore = require('connect-session-knex')(session);
+
+const apiRouter = require('./api-router');
+const knex = require('../data/dbConfig');
+
+const server = express();
+
+const sessionConfiguration = {
+    name: 'snickerdoodle',
+    secret: 'keep it secret, keep it safe',
+    saveUninitialized: true,
+    resave: false,
+
+    // store session
+    store: new KnexSessionStore({
+        knex,
+        tablename: 'session',
+        createTable: true,
+        sidfieldname: 'sid',
+        clearInterval: 1000 * 60 * 10,
+    }),
+
+    // cookie options
+    cookie: {
+        maxAge: 1000 * 60 * 10,
+        secure: false,
+        httpOnly: true,
+    }
+}
```

Comment:
This would be a good area to think about separating out into it;s own file. If you use cookies in a personal project, play around with the idea